src/utils/aadGroup.ts

```diff
@@ -33,6 +34,7 @@ export const aadGroup = {
   /**
    * Get a single group by its display name.
    * @param displayName Group display name.
+   * @param selectProperties Properties to select if specified.
```

Comment:
Is there a reason why this description was added?

Reply:
I used this method before. My bad